### ISSUE TITLE
after auto-update to version 1.0.0 > directory and storage customization is not retained

### ISSUE THREAD

**Issue by betokraut** [noia-network/noia-node-gui#61]
**Describe the bug**

On Windows 10 after an auto-update to version 1.0.0 from previous version:
1) the wallet address reverts to blank after inputting address and saving
2) the storage directory and size input reverts to default after saving

**To Reproduce**

1) under wallet tab: pasting in wallet address with ctrl+v or using 'paste from clipboard' button and then saving
2) under settings tab: changing folder to directory on local C: drive and setting size to 10gb

**Expected behavior**

1)+2) inputs are being saved and displayed after reconnecting

**Info:**

-   OS: Windows 10
-   Version 1.0.0

**Additional context**

Thank you

**Comment by durbanite** [comment]
I am experiencing the same issue, all settings are removed after the node restarts

**Comment by energoico** [comment]
I am experiencing the same issue!

**Comment by Dancia** [comment]
Does this happen if the directory is `C:\` or with any directory, for example `C:\storage`?

**Comment by betokraut** [comment]
Hey Dancia, just tested it again, the answer to your question would be: any directory, thanks.

**Comment by Dancia** [comment]
I have another question which would greatly help us debugging this issue.

By default, NOIA node saves it's settings file in directory: `C:\Users\YourUserName\AppData\Roaming\noia-node`, where `YourUserName` is your Windows account name. Settings file is named `node.settings`. Please try opening this settings location and NOIA node GUI side-by-side and perform this test:
1. Note `node.settings` `Data modified` timestamp.
2. Click `Save` button in GUI.
3. Check if `node.settings` `Data modified` has changed.

![image 2](https://user-images.githubusercontent.com/8902599/48778917-1a6ccc00-ecdf-11e8-8621-93d582a354f6.png)

After pressing `Save` button `Date modified` date should have changed and `node.settings` should contain update settings. You can investigate `node.settings` file content too.

Please report if `node.settings` content changes and/or `Date modified` value.

Note: If you fail to find `node.settings` location, you can also find it by opening console (WinKey+R, write `cmd` and press `OK`), enter in console window your NOIA installation location: `"C:\Users\YourUserName\AppData\Local\Programs\noia-node-gui\NOIA.exe"` and press enter. You should see something like this:
![image 3](https://user-images.githubusercontent.com/8902599/48779117-90713300-ecdf-11e8-9654-4ef3d134d877.png)
Where `Initializing NOIA node, settings-path=...` will print settings location.

**Comment by haianh1610** [comment]
Hi,
I have done as your request and here is the results:
- Data Modified value changed after clicking Save button.
- Storage size is reverted to the previous setting value.

One more issue: I successfully saved the wallet address then came to Settings to change the storage size. After clicking Save button, a message showed "settings updated", after that the storage size was still zero and the wallet address was cleared. I had to repeat disconnecting node, pasting wallet address, resetting node...in around 20 minutes to save the wallet address again. I still have 3 more nodes with 0B storage size but i don't dare to change the settings due to this issue. 

![noia 1](https://user-images.githubusercontent.com/45105049/48792935-1a44ee80-ed28-11e8-9b7e-f5bf335fa1f2.jpg)
![noia 2](https://user-images.githubusercontent.com/45105049/48792936-1a44ee80-ed28-11e8-90d2-e174bc706f82.jpg)

**Comment by betokraut** [comment]
Hey Dancia,
1) the issue persisted when running as admin 
2) the timestamp of 'note.settings' was successfully changed whenever the save button was clicked
Thanks

**Comment by martijnspaan** [comment]
I also encounter this issue with version 1.0.0 and just did a fresh install of 1.0.1 but still my settings are not stored. I changed the node.settings file manually with notepad and restarted the client. The settings are correctly set now and just checked with Nojus that my node is online. So that would be a workaround for now.

**Comment by durbanite** [comment]
That is what I did when the new version came out, Nojus confirmed that my
node was connected. I also ran the exe with admin rights, if that helps.

**Comment by martijnspaan** [comment]
Just did some further investigation and found something interesting. Using Process Monitor of Sysinternals I noticed that when pressing the save button, the settings file is created (20:46:50) but not written. When I open the file it is completely empty. Then the server begins to restart and when that is done, a different process id (2564 instead of 7524) is also writing the settings file, but clearly it uses default values because the settings file is empty.

![2018-11-20 20_47_12-ttt - 52 174 44 233_3389 - remote desktop connection](https://user-images.githubusercontent.com/6927615/48798939-2c159a00-ed06-11e8-8617-0056af12ea66.png)

**Comment by betokraut** [comment]
Thank you Martijnspaan. Changing the 'node.settings' manually worked for the 'storage directory' and 'storage size'. What is the correct syntax for setting the 'storage size' to 10GB? I seem to be stuck with byte size only and I can't set it to above 1000B. 
![storage-size](https://user-images.githubusercontent.com/45111926/48800483-1f258600-ecc7-11e8-9766-d9da3c34f86b.jpg)

**Comment by martijnspaan** [comment]
@betokraut you just changed your port number to 1000 instead of the storage size 😂 

Also be carefull posting your screenshots, your nodeId, username and part of the wallet mnemonic are showing, not sure if that can be a privacy / security issue.

Here are the settings you can change:
![2018-11-20 21_27_08-ttt - 52 174 44 233_3389 - remote desktop connection](https://user-images.githubusercontent.com/6927615/48800941-72b9c300-ed0b-11e8-8e08-f6f16e13a1f6.png)

**Comment by betokraut** [comment]
@martijnspaan jaja, silly me+ most grateful
![storage-size](https://user-images.githubusercontent.com/45111926/48801685-416cd300-ecca-11e8-9f2a-e488dfecbd0a.jpg)

**Comment by Tartraus** [comment]
I have the same prob. on my Mac Os, after latest update and downloaded it a couple days later today & tried again? Wallet Address is there this time! 
On NODE Page  shows only my amount of set GB and the time moving, the only movement? I pressed on check port -
WebRTC control port 
8048
Check port
Connection timed out
Might have to dumb it down for me? ":)
Thanks!
